svelte-typescript-app/cypress/e2e/todolist.cy.js

```diff
@@ -12,68 +13,68 @@ describe("Doaa", function () {
   });
 
   it("create item-normal", () => {
-    cy.get("#inputID").type("88");
-    cy.get("#inputTask").type("philo's test");
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("88");
+    homepage.getContentTextBox.type("philo's test");
+    homepage.getCreateBtn.click();
```

Comment:
1. You should only call one func instead of these three lines like `homepage.CreateNewItem(ID,Text)` and in homapage class make this func and put these three lines.
2.  Always use parameter not hard coded variables like `text` instead of "philo's test"
3.  Now you have created a new item but where is the assertions to make sure that this item is created properly.